code/ATMOSPHERICS/components/unary/vent_scrubber.dm

```diff
@@ -2,6 +2,8 @@
 	icon = 'icons/atmos/vent_scrubber.dmi'
 	icon_state = "map_scrubber"
 
+	req_one_access_txt = "24;10"

```

Comment:
Multitool menus aren't shit like hacking.

They're configuration menus for machinery such as Tcomms